odfdom/src/test/java/org/odftoolkit/odfdom/doc/table/TableCellTest.java

```diff
@@ -270,12 +272,11 @@ public void testSetGetFormat() throws Exception {
     OdfTableCell cell = tablerow.getCellByIndex(3);
     Calendar currenttime = Calendar.getInstance();
     cell.setDateValue(currenttime);
-    cell.setFormatString("yyyy-MM-dd");
+    cell.setFormatString("YYYY-MM-dd");
```

Comment:
use `yyyy` here, `YYYY`is the week-based year, which will give wrong results in the first week of january if the week starts in the year before.